019-ThisOldDataWarehouse/Coach/Solutions/Challenge5/README.md

```diff
@@ -29,7 +29,7 @@ The objective of this lab is to have the Power BI report "WWI_Sales.pbix" to ret
 |                                            |                                                                                                                                                       |
 | ------------------------------------------ | :---------------------------------------------------------------------------------------------------------------------------------------------------: |
 | **Description**                            |                                                                       **Links**                                                                       |
-| Performance Tuning | <https://medium.com/@ian.shchoy/azure-sql-data-warehouse-deep-dive-into-data-distribution-f4cf8f1e340a> |
+| Performance Tuning | https://medium.com/@ian.shchoy/azure-sql-data-warehouse-deep-dive-into-data-distribution-f4cf8f1e340a |
```

Comment:
This link will not render on some MD renderers.... you should put the angle brackets back, like the others.